vcluster/_fragments/integrations/istio.mdx

```diff
@@ -57,38 +58,39 @@ This configuration:
 Only `DestinationRules`, `Gateways`, and `VirtualServices` from `networking.istio.io/v1` API Version are synced to the host clusters. Other kinds are not yet supported.
 :::
 
-### Set up cluster contexts
-
-Setting up the host and virtual cluster contexts makes it easier to switch
-between them.
-
-```bash
-export HOST_CTX="your-host-context"
-export VCLUSTER_CTX="vcluster-ctx"
-export VCLUSTER_HOST_NAMESPACE="vcluster"
-export GATEWAY_API_VERSION="v1.2.1" # set this according to your Istio version / GatewayAPI version matrix above
-```
-
-:::tip
-You can find your contexts by running `kubectl config get-contexts`
-:::
-
 <!--vale off-->
 ## Route request based on the version label of the app
-
 <Flow id="istio-example">
 
+  ### Set up cluster contexts
+  <Step>
+  Setting up the host and virtual cluster contexts makes it easier to switch
+  between them. You can adjust the values below according to your setup. The 
+  commands in the following steps will be automatically updated to follow your 
+  configuration. For the GatewayAPI version, you should set it according to 
+  your Istio version / GatewayAPI version matrix above.
+
+  <PageVariables HOST_CONTEXT="your-host-context" VCLUSTER_CONTEXT="vcluster-ctx" VCLUSTER_HOST_NAMESPACE="vcluster" GATEWAY_API_VERSION="v1.2.1" TEST_NAMESPACE_NAME="test"/>
```

Comment:
Non-blocking comment: maybe we could explain what is HOST_CONTEXT or find a better variable name